modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/resources/com/liferay/dynamic/data/mapping/form/field/type/packageinfo

```diff
@@ -1 +1 @@
-version 3.0.0
```

Comment:
@p-albuquerque why do we have baseline in this PR?

Reply:
I think this is not related with my changes, but in my preliminar CI results [here](https://testray.liferay.com/home/-/testray/case_results?tab=runs&testrayBuildId=2552833052), this versioning error was [caught](https://testray.liferay.com/reports/production/logs/2024-03/test-1-32/test-portal-acceptance-pullrequest(master)/4008/modules-semantic-versioning-jdk8/0/0/jenkins-console.txt.gz?authuser=0).